cmd/version/version.go

```diff
@@ -60,23 +66,62 @@ func getVersion(ctx context.Context, args []string, cmdCtx cmdCore.CommandContex
 		return err
 	}
 
+	getControlePlaneVersion(ctx, cmdCtx)
+	return nil
+}
+
+func printVersion(response versionOutput) error {
+	b, err := json.MarshalIndent(response, "", "  ")
+	if err != nil {
+		return fmt.Errorf("err %v: ", err)
+	}
+	fmt.Print(string(b))
+	return nil
+}
+
+func compareVersion(latest, current string) string {
+	latestVersion, _ := hversion.NewVersion(latest)
+	fversion, _ := hversion.NewVersion(current)
+
+	if fversion.LessThan(latestVersion) {
+		return fmt.Sprintf("A newer version of flytectl is available [%v] Please upgrade using - https://docs.flyte.org/projects/flytectl/en/latest/index.html", latest)
```

Comment:
We should think of having a single step upgrade for the user by just giving him a prompt if he wants to upgrade 

check this out  https://equinox.io/ and see if its possible to use it.

Reply:
Yes, I know about it and it's amazing if we can use it. You can create an issue for that